docs/syntax.md

```diff
@@ -291,7 +291,7 @@ class Foo
 
   def foo: () -> instance
```

Comment:
@soutaro I'm also surprised by how it's used in this case? What's the meaning of it? Is it the same as `self`?

Reply:
Dummy names like ‘foo’ don’t always make comprehensible general examples. 👍

Reply:
@Morriar This is in fact same with `self`. They are different on `def self?.foo: `, but yeah, it's exactly same in this case.